content-src/asrouter/asrouter-content.jsx

```diff
@@ -280,6 +284,21 @@ export class ASRouterUISurface extends React.PureComponent {
     return null;
   }
 
+  renderTrailhead() {
+    const {message} = this.state;
+    if (message.template === "trailhead") {
+      return (<Trailhead
+        document={this.props.document}
+        message={message}
+        onAction={ASRouterUtils.executeAction}
+        onDoneButton={this.dismissBundle(this.state.bundle.bundle)}
```

Comment:
just realized this isn't used in `Trailhead`. Probably fine. Looks like it's to dismiss a bundle of cards but not sure why as they won't appear again anyway.